recipes/logzero/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "logzero" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "f3433f87a184e1d846099c521437bfe9d463d4b74c3b2da3488b4f52ce821a1d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - colorama
+
+test:
+  imports:
+    - logzero
+
+about:
+  home: https://github.com/metachris/logzero
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Python logging made easy'
+  description: |
+    Robust and effective logging for Python 2 and 3.
+  doc_url: https://logzero.readthedocs.io/
+  dev_url: https://github.com/metachris/logzero
+
+extra:
+  recipe-maintainers:
+    - carlodri
+    - metachris
```

Comment:
@metachris are you ok being added as a co-maintainer on the `logzero` recipe on conda-forge?

https://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role

Reply:
@metachris [has been warned](https://github.com/metachris/logzero/issues/67).... 😉

Reply:
Yes! Thanks for the support.